### PR TITLE
feat: Add utilitarian docker compose to run external services locally

### PR DESCRIPTION
**Pull request by fzavalia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,4 +65,32 @@ contents
...
+
+First, run all services simultaneously with:
+
+`docker-compose up -d`
+
+`-d or --detach` will run the processes on the background instead of running on the terminal which executed the command. 
+You can ignore this flag if you don't care about it.
+
+Before running the `builder-server`, make sure that the following `.env` variables are set correctly so the services run by docker-compose
+are consumed correctly.
```
Comment (LautaroPetaccio) [inline README.md:84]: Is that what you meant?
```suggestion
work as expected.
```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Exposes endpoints for the Builder.
 
-## RUN
+## <a name="run"></a>RUN
```
[the remark this answers is not in the document]
Comment (fzavalia) [reply README.md:5]: I did it so I could redirect the user to the RUN section from this part. Is there another way?
https://github.com/decentraland/builder-server/pull/297/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R94

File: README.md (hunk 1 above)
Comment (fzavalia) [reply README.md:84]: Sounds better

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (fzavalia) [reply README.md]: It actually works without the anchor
https://gist.github.com/rachelhyman/b1f109155c9dafffe618
Nice!

**Pull request merged**